libs/build.gradle

```diff
@@ -45,4 +45,16 @@ configure(childProjects.values()) {
      */
     apply plugin: 'elasticsearch.build'
   }
+
+  // This is for any code potentially included in the server at runtime.
```

Comment:
Can we just add this to the plugin-scanner project for now. In general we want to move more configuration where it belongs. adding code that might be needed in the future in sometime just adds clutter for now.

Reply:
It needs to apply to every lib _except_ plugin-scanner.

Reply:
(Really, plugin-scanner shouldn't be in `libs` in the first place.)